arch/arm64/boot/dts/qcom/msm8953-lenovo-kuntao.dts

```diff
@@ -47,6 +71,25 @@
 		};
 	};
 
+	i2c-sensors {
+		compatible = "i2c-gpio";
+		sda-gpios = <&tlmm 14 (GPIO_ACTIVE_HIGH|GPIO_OPEN_DRAIN)>;
+		scl-gpios = <&tlmm 15 (GPIO_ACTIVE_HIGH|GPIO_OPEN_DRAIN)>;
+		i2c-gpio,delay-us = <2>; /* ~100 kHz */
+		#address-cells = <1>;
+		#size-cells = <0>;
+
+		imu@6a {
+			compatible = "st,lsm6ds3";
+			reg = <0x6a>;
+			vdd-supply = <&pm8953_l22>;
+			vddio-supply = <&pm8953_l6>;
+			mount-matrix = "0", "-1", "0",
+						"-1", "0", "0",
```

Comment:
```suggestion
					 "-1", "0", "0",
```

Reply:
Not sure how do I align? How Many tabs it needs?

Reply:
You can check alignment with `git diff`. tbh i don't like tab indentation because this should solved with mixing tabs and space you will need couple of tab until it is not over the first line of the matrix (after =) and the rest of the padding should be solved with 1 or 2 spaces.